tools/sc64/README.md

```diff
@@ -1,14 +1,5 @@
-# Source
-https://github.com/Polprzewodnikowy/SummerCart64
-
-# License
-GPL-3.0 license
```

Comment:
Has the license changed?

Reply:
Although, I guess we are not actually using anything from it, so it falls into fair use.